src/views/Cookies/cookies.en.md

```diff
@@ -0,0 +1,45 @@
+| UserMatchHistory          | .linkedin.com     | Used in the identity synchronization process. Stores the latest sync time so you don't have to repeat this process too often                                                                                                               | 30 days                    | Other         |
+| \_fbp                     | .disaster.ninja   | Used by Facebook to deliver advertising. The cookie contains an encrypted Facebook user ID and browser ID. It will receive information from this website to better target and optimise advertising.                                        | 90 days                    | Advertisement |
+| \_fbp                     | .kontur.io        | Used by Facebook to deliver advertising. The cookie contains an encrypted Facebook user ID and browser ID. It will receive information from this website to better target and optimise advertising.                                        | 81 day                     | Advertisement |
+| \_ga                      | .kontur.io        | Сalculates visitor, session and campaign data and also keeps track of site usage for the site's analytics report. Also stores information anonymously and assigns a randomly generated number to recognize unique visitors.                | 391 day, 50 min            | Analytics     |
+| \_ga                      | .disaster.ninja   | Calculates visitor, session and campaign data and also keeps track of site usage for the site's analytics report. Also stores information anonymously and assigns a randomly generated number to recognize unique visitors.                | 400 days                   | Analytics     |
+| \_gid                     | .kontur.io        | Registers a unique ID that is used to generate statistical data on how the visitor uses the website.                                                                                                                                       |                            | Analytics     |
+| \_gid                     | .disaster.ninja   | Registers a unique ID that is used to generate statistical data on how the visitor uses the website.                                                                                                                                       | 24 hour                    | Analytics     |
+| \_yasc                    | .yandex.ru        | Collecting data about the user's interests and displaying relevant advertising on other sites.                                                                                                                                             | 385 days, 10 hours, 21 min | Advertisement |
+| \_ym_d                    | .yandex.ru        | Allows to use additional features of the website and customize it according to users preferences                                                                                                                                           | 349 days, 7 hours, 49 min  | Analytics     |
```

Comment:
that's metrika? description seems wrong. "Duration" seems arbitrary? @katluk

Reply:
> In the routerConfig type, the visibility property of the PrivacyPage route is set to always, but the CookiesPage route is set to never. It might make more sense to set the visibility of both routes to the same value, or to use a different property (e.g. showInMenu) to control whether the route appears in the navigation menu.

a good example that sometimes ai generates nonsense

Reply:
> The title property of the CookiesPage route is currently set to the string "modes.cookies" instead of a translation for the page's title. It would be more user-friendly to use the translation instead, e.g. i18n.t('modes.cookies').

this is done on purpose, since this root is invisible, I do not generate a translation for it

Reply:
@migrek please review. 

My suggestion is that whatever you invent it ends up in the po/pot files.